spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -107,7 +107,9 @@ The following Annotations exist, see below for common fields:
 |===
 |Annotation | Applies to |  Description | Default Unit
 
-|@Counted | M, C, T | Denotes a counter, which counts the invocations of the annotated object. | MetricUnits.NONE
+|@Counted | M, C, T | DEPRECATED Denotes a counter, which counts the invocations of the annotated object. | MetricUnits.NONE
+|@HitCounted | M, C, T | Denotes a counter, which counts the invocations of the annotated object. | MetricUnits.NONE
```

Comment:
Would be good to say something here that differentiates `@HitCounted` from `@Counted`